### PR TITLE
Add Barcelona Duo final blog post and images.

### PR DESCRIPTION
**Pull request by amaliacardenas**
Co-Authored-By: loriking <loriannking@gmail.com>

### CODE REVIEW

File: blog/_posts/2018-09-20-team-barcelonaduo-goodbye.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+title: The end of the beginning
+layout: post
+created_at: Thurs September 20 2018
+permalink: blog/2018-09-20-team-barcelonaduo-goodbye
```
Comment (v-thakkar) [inline blog/_posts/2018-09-20-team-barcelonaduo-goodbye.md]: Hi @amaliacardenas ! Thanks for the PR. The blog post looks amazing. The only comment I have is all images are missing image-credits and image description. Can you please add them? We'll be ready to publish it after that. :)

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2018-09-20-team-barcelonaduo-goodbye.md (hunk 1 above)
Comment (amaliacardenas) [reply blog/_posts/2018-09-20-team-barcelonaduo-goodbye.md]: Hi @nerdyvaishali. I added the missing image credits and image description. Let me know if there is anything else that needs to be added. :)

**Review by v-thakkar — APPROVED** [review]

File: blog/_posts/2018-09-20-team-barcelonaduo-goodbye.md (hunk 1 above)
Comment (v-thakkar) [reply blog/_posts/2018-09-20-team-barcelonaduo-goodbye.md]: Hi @amaliacardenas ! Your pull request contains 2 blogpost files and bunch of unnecessary images. Can you please remove unnecessary images and file? Thanks

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by amaliacardenas** [comment]
Hi @nerdyvaishali 
Thank you for your comments. As requested, I removed the duplicate blog post and deleted the duplicate images. :)

**Review by v-thakkar — APPROVED** [review]

**Pull request merged**